src/DonationForms/ViewModels/DonationConfirmationReceiptViewModel.php

```diff
@@ -166,7 +163,7 @@ public function enqueueGlobalStyles(string $primaryColor, string $secondaryColor
      *
      * @return void
      */
-    private function enqueueFormScripts(int $formId, string $formDesignId)
+    private function enqueueFormScripts(?string $formDesignId)
```

Comment:
We should add an unreleased tag noting the change in signature.

Reply:
Thanks, this is not actually released yet (still 3.0.0)

Reply:
Ok, then disregard 😎